gangsta.js

```diff
@@ -17,8 +19,58 @@
         random = function (items) {
             return items[randmax(items.length)];
         },
+        swapChar = function (pwChar, lChars) {
+            var c, 
```

Comment:
If we use the character ("A") for the keys, and we pass swapChar the character, we can do something like this:

```javascript
    if (leetChars.indexOf(pwChar.toUpperCase()) === -1) {
        return pwChar;
    }
    ...
```